static/js/composebox_typeahead.js

```diff
@@ -88,6 +88,35 @@ var nextFocus = false;
 function handle_keydown(e) {
     var code = e.keyCode || e.which;
 
+    if ((code === 66 || code === 73 || code === 75) && (e.ctrlKey || e.metaKey)) {
+        var textarea = $("#new_message_content");
+        var range = textarea.range();
+        function add_markdown(markdown) {
+        if (!document.execCommand('insertText', false, markdown)) {
+            textarea.range(range.start, range.end).range(markdown);
+            }
+        }
+
+        if (range.length > 0) {
+            if (code === 66) {
+                // convert selected text to bold text
```

Comment:
I'd expand this comment to make clear what the key is, e.g.
// ctrl+b: Convert selected text to bold text

Reply:
@YJDave, @timabbott I think we can remove the  `if (range.length > 0)` thing here so it becomes - 
<img src="https://d26dzxoao6i3hh.cloudfront.net/items/292A1l0V250E352w433X/Screen%20Recording%202017-11-24%20at%2003.49%20PM.gif?v=59d75677">

This would then matches with GitHub's shortcuts behaviour.

Reply:
If we remove this condition then it will override other keyboard shortcuts like `Ctr+shift+i` is to enable developer tools. So if user will press `Ctr+shift+i` then it will enable developer tools and also paste `**` into compose box.